server/views/pages/profile.html

```diff
@@ -103,7 +103,7 @@ <h2 class="govuk-heading-m">Today's timetable</h2>
     <div class="govuk-width-container">
       <h2 class="govuk-heading-m">Incentive level</h2>
       {% if incentivesSummary.error %}
-        <p class="govuk-!-font-size-24" data-test="incentives-error">We are not able to show your incentive level at this time. <a href="{{ params.returnUrl }}" class="govuk-link">try again</a> or <a href="{{ incentiveSummary.link }}" class="govuk-link">find out more about incentive levels</a>.</p>
+        <p class="govuk-!-font-size-24" data-test="incentives-error">We are not able to show your incentive level at this time. <a href="{{ params.returnUrl }}" class="govuk-link">Try again</a> or <a href="{{ incentiveSummary.link }}" class="govuk-link">find out more about incentive levels</a>.</p>
```

Comment:
Content incentives error link needs updating from my original commit

{{ incentivesSummary.link }}

Reply:
Good spot, fixed that